src/features/surveys/components/SurveySubmissionsList.tsx

```diff
@@ -124,10 +125,12 @@ const SurveySubmissionsList = ({
 
     if (row.respondent?.id) {
       return (
-        <ZUIPersonGridCell
-          onClick={startEditing}
-          personId={row.respondent.id}
-        />
+        <ZUIPersonHoverCard personId={row.respondent.id}>
```

Comment:
Nice solution! Another option would have been to put the `ZUIPersonHoverCard` inside the `ZUIPersonGridCell` so that it would also work for views, but on the other hand that might be error-prone, so I like this. 💯